h/templates/admin/cohorts.html.jinja2

```diff
@@ -0,0 +1,49 @@
+{% extends "h:templates/layouts/admin.html.jinja2" %}
+
+{% set page_id = 'cohorts' %}
+{% set page_title = 'Feature Cohorts' %}
+
+{% block content %}
+  <p>
+    On this page you can see a list of all the feature cohorts.
+  </p>
+  <p>
+    Back to <a href="{{ request.route_url('admin_features') }}">Feature flags</a>
```

Comment:
It would be nice if we could extend our use of the bootstrap navigation bar rather than adding some ad-hoc links to the page content like this. See https://getbootstrap.com/components/#navbar for some examples -- and perhaps @robertknight for some help if you're not sure what to do.